GUI/CKAN-GUI.csproj

```diff
@@ -233,7 +239,8 @@
       <SubType>Component</SubType>
     </Compile>
     <Compile Include="Controls\TriStateToggle.cs">
-      <SubType>Component</SubType>
+      <SubType>            TimeLog t = new TimeLog(CurrentInstance.CkanDir());
```

Comment:
Looks like an accidental paste of code where this should just say `Component`.

Reply:
Ok i've fixed it